documentation/src/main/asciidoc/topics/server_protocols_rest.adoc

```diff
@@ -463,3 +463,93 @@ public class RestExample {
 }
 
 ----
+
+[[rest_server_client_java]]
+=== Java11+
+
+[source,java]
+----
+
+package org.infinispan;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.Base64;
+
+/**
+ * Rest example accessing a cache using the new HttpClient API from Java 11 and newer.
```

Comment:
RestExample class shows you how to access your cache via HttpClient API with Java 11 or later.